chef/cookbooks/ceph/recipes/conf.rb

```diff
@@ -62,6 +62,9 @@
 # calculate pg_num based on documentation
 # http://ceph.com/docs/master/rados/operations/placement-groups/
 case num_osds
+when 0
+  Chef::Log.warn("Ceph recipe invoked but there are no OSDs!")
+  return
```

Comment:
right, I understand, but do we want to still create the ceph.conf and anything else happening afterwards still in this case? sounds like logging the error and then defaulting to a default pg_num might also be an option. 

Not sure if it really matters though (I'm not torn on this, just wanted to raise the thought as I had it)